lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Librarian.java

```diff
@@ -0,0 +1,23 @@
+package com.codedifferently.lesson10.library;
+
+import com.codedifferently.lesson10.library.exceptions.BookCheckedOutException;
+
+public class Librarian {
```

Comment:
Take a look at my example in Lesson 12 and see if you can adapt to what you have here.